examples/seismic/acoustic/acoustic_example.py

```diff
@@ -70,25 +70,22 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
                         help="Preset to determine the number of dimensions")
     parser.add_argument('-f', '--full', default=False, action='store_true',
                         help="Execute all operators and store forward wavefield")
-    parser.add_argument('-a', '--autotune', default='off',
-                        choices=(configuration._accepted['autotuning']),
-                        help="Operator auto-tuning mode")
     parser.add_argument("-so", "--space_order", default=6,
                         type=int, help="Space order of the simulation")
     parser.add_argument("--nbl", default=40,
                         type=int, help="Number of boundary layers around the domain")
     parser.add_argument("-k", dest="kernel", default='OT2',
                         choices=['OT2', 'OT4'],
                         help="Choice of finite-difference kernel")
-    parser.add_argument("-dse", default="advanced",
-                        choices=["noop", "basic", "advanced", "aggressive"],
-                        help="Devito symbolic engine (DSE) mode")
-    parser.add_argument("-dle", default="advanced", choices=["noop", "advanced"],
-                        help="Devito loop engine (DLE) mode")
     parser.add_argument("--constant", default=False, action='store_true',
                         help="Constant velocity model, default is a two layer model")
     parser.add_argument("--checkpointing", default=False, action='store_true',
                         help="Constant velocity model, default is a two layer model")
+    parser.add_argument("-opt", default="advanced", choices=["noop", "advanced"],
+                        help="Performance optimization level")
+    parser.add_argument('-a', '--autotune', default='off',
```

Comment:
@georgebisbas  this is still brokento pass a list right? like `-a advanced,runtime`

Reply:
Yes, I think so.

Reply:
Ok, just checking thanks